modules/tinymce/src/themes/silver/test/ts/browser/window/SilverInlineDialogTest.ts

```diff
@@ -24,8 +24,7 @@ describe('browser.tinymce.themes.silver.window.SilverInlineDialogTest', () => {
 
   const dialogSize: Dialog.DialogSize = 'normal';
 
-  const createDialogSpec = (size?: Dialog.DialogSize, dummy: boolean = false): Dialog.DialogSpec<{ fred: string }> & { size?: Dialog.DialogSize } => {
-
+  const createDialogSpec = (size?: Dialog.DialogSize, dummy: boolean = false, iframe: boolean = false): Dialog.DialogSpec<{ fred: string }> & { size?: Dialog.DialogSize } => {
```

Comment:
Adding an iframe is basically a whole new dailog spec. Should we just create a separate dailogWithIfram spec?

Reply:
yes

Reply:
done